libs/ui/form-field/helm/src/lib/hlm-form-field.component.ts

```diff
@@ -23,13 +22,13 @@ import { HlmErrorDirective } from './hlm-error.directive';
 	},
 })
 export class HlmFormFieldComponent {
-	control = contentChild(BrnFormFieldControl);
+	private readonly control = contentChild(BrnFormFieldControl);
```

Comment:
should these be private? I feel like people might want to access them programatically?

Reply:
yes probably better to change to public readonly